rules/windows/process_creation/proc_creation_win_using_set_service_to_hide_services.yml

```diff
@@ -0,0 +1,35 @@
+title: Abuse of Service Permissions to Hide Services Via Set-Service
+id: 514e4c3a-c77d-4cde-a00f-046425e2301e
+related:
+    - id: a537cfc3-4297-4789-92b5-345bfd845ad0
+      type: derived
+status: experimental
+description: Detects usage of the "Set-Service" powershell cmdlet to configure a new SecurityDescriptor that allows a service to be hidden from other utilities such as "sc.exe", "Get-Service"...etc. (Works only in powershell 7)
+author: Nasreddine Bencherchali
+references:
+    - https://twitter.com/Alh4zr3d/status/1580925761996828672
+    - https://learn.microsoft.com/en-us/powershell/module/microsoft.powershell.management/set-service?view=powershell-7.2
+date: 2022/10/17
+logsource:
+    category: process_creation
+    product: windows
+detection:
+    selection_img:
+        - Image|endswith: '\pwsh.exe'
```

Comment:
not powershell.exe  or powershell_ise.exe too ?

Reply:
To my knowledge The SDDL flag in the cmdlet `set-service` is available only in PowerShell 7 which is pwsh.exe